src/DonationForms/V2/Endpoints/ListDonationForms.php

```diff
@@ -128,10 +128,21 @@ public function handleRequest(WP_REST_Request $request): WP_REST_Response
             $this->listTable->items($forms, $this->request->get_param('locale') ?? '');
             $items = $this->listTable->getItems();
 
-            foreach($items as &$item ) {
+            foreach ($items as &$item) {
+                $v2form = defined('GIVE_NEXT_GEN_VERSION') && ! get_post_meta($item['id'], 'formBuilderFields');
+                $migrate = $v2form
+                           && ! get_post_meta($item['id'], 'migratedFormId');
+
+                $transfer = $v2form
+                            && get_post_meta($item['id'], 'migratedFormId')
+                            && ! get_post_meta($item['id'], 'transferredFormId');
```

Comment:
Similar to the `migratedFormId` the reference for the `transferredFormId` is stored on a corresponding v3 form.